### PR TITLE
Fix cabal new-build and stack build

### PR DESCRIPTION
**Pull request by denisshevchenko**
Now it can compiles with `cabal new-build`.

### CODE REVIEW

File: app/TxSubmission.hs [hunk 1]
```diff
@@ -15,13 +15,13 @@ module TxSubmission (
...
-import           Data.Void (Void)
 import           Data.ByteString.Lazy (ByteString)
+import           Data.Proxy
 import qualified Data.Set as Set
+import           Data.Void (Void)
 import           Options.Applicative
-import           Data.Proxy
 
-import qualified Codec.Serialise as Serialise (encode, decode)
+import qualified Codec.Serialise as Serialise (decode, encode)
```
Comment (cleverca22) [inline app/TxSubmission.hs]: order was swapped

File: app/TxSubmission.hs [hunk 2]
```diff
@@ -15,13 +15,13 @@ module TxSubmission (
     , localSocketAddrInfo
     ) where
 
-import           Data.Void (Void)
 import           Data.ByteString.Lazy (ByteString)
+import           Data.Proxy
 import qualified Data.Set as Set
+import           Data.Void (Void)
```
Comment (cleverca22) [inline app/TxSubmission.hs]: void was moved

File: app/TxSubmission.hs [hunk 3]
```diff
@@ -15,13 +15,13 @@ module TxSubmission (
...
 import           Options.Applicative
-import           Data.Proxy
```
Comment (cleverca22) [inline app/TxSubmission.hs]: proxy was moved

File: cardano-node.cabal [hunk 4]
```diff
@@ -24,7 +24,7 @@ library
                        base >=4.12 && <5
                      , bytestring
                      , canonical-json
-                     , cborg
+                     , cborg == 0.2.2.0
```
Comment (mrBliss) [inline cardano-node.cabal]: Do we want `==` or `>= && <`?

**Pull request merged**